Cargo.toml

```diff
@@ -21,6 +21,8 @@ concurrent-queue = "2.0.0"
 fastrand = "2.0.0"
 futures-lite = { version = "2.0.0", default-features = false }
 slab = "0.4.4"
+thread_local = "1.1"
+atomic-waker = "1.0"
```

Comment:
Please keep the dependencies alphabetized.

Reply:
Done.